packages/types/src/page/extended.d.ts

```
@@ -0,0 +1 @@
+export type TPageExtended = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)